test/bower-fixtures/custom-package/bower.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I'd rename this one from `custom-package` to `custom-package-not-ok` and create another test, named `custom-package-ok` that will have all correct packages.

Otherwise we only know the module will fail with invalid packages but we don't know if it succeeds for valid setup. :)